### ISSUE TITLE
[SF][A11y][WCAG2.2][Link]: High contrast mode not fully adapting with system contrast setting

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#9826]
### Bug Description

hi, team

Recently, I facing the issue about "High contrast mode not fully adapting with system contrast setting"

After enabling High Contrast mode through the BizX or application settings, the screen properly adapts to black on white or white on black themes. However, when the High Contrast mode is applied through the system configuration, the screen content does not fully adapt. Links display in different colors, comment flyout outlines not visible and link colors dynamically change depending on the chosen high contrast theme, such as Aquatic, Desert, or Night Sky.

### Expected Behaviour

1. The screen's background colour should adjust according to the selected high-contrast theme, such as switching to black or white background colours, depending on the chosen theme.
2. The application should consistently adapt to both system and application theme changes.
3. All elements and keyboard focus colors, should adopt both themes consistently

 

### Steps to Reproduce

1. Apply application-level high contrast setting -  High Contrast White Text on Black Background 
2. Switch to one of the high-contrast themes from the Operating system
On Windows 11, choose one of Aquatic, Dusk, or Night Sky

We will get the below: 
<img width="1491" alt="image" src="https://github.com/user-attachments/assets/5fd8f376-0c50-4811-978c-d22f54167e53">

### Priority

Medium

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by didip1000** [comment]
Hey @SAP/ui5-webcomponents-topic-core,

This is the same issue as https://github.com/SAP/ui5-webcomponents/issues/9436, but for version 1.24.

Thank you,
Diana P.

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh #9436 looks similar, could you check what has been suggested in the [last comment](https://github.com/SAP/ui5-webcomponents/issues/9436#issuecomment-2437582350).
